### DISCUSSION TITLE
How to disable rumble ?

### DISCUSSION THREAD

**Discussion by SpyErx** [nefarius/DsHidMini#207] [Q&A] [answered]
As the title says. Was previously using SCP toolkit with my ps3 controller and it was very easy to disable it in options but can't seem to find any way to disable it in dshidmini. Read through FAQ as there was no mention on how to do it so hopefully you actually can.

**Comment by nefarius** [comment, marked as answer]
You can't. It's in the works, see #197

**Reply by SpyErx** [reply]
Thanks for reply, guess i'll stick with SCP toolkit until then.
